enterprise/utils.py

```diff
@@ -636,3 +636,13 @@ def get_program_type_description(program_type):
     :return: The description associated with the program type. If none exists, then the empty string.
     """
     return PROGRAM_TYPE_DESCRIPTION.get(program_type, '')
+
+
+def get_enterprise_utm_context(enterprise_customer):
+    """
+    Get the UTM context for the enterprise.
+    """
+    return {
+        'utm_medium': 'enterprise',
+        'utm_source': enterprise_customer.name
```

Comment:
We should call `slugify` here to format the EntepriseCustomer name as a query parameter value like @saleem-latif  is doing [here](https://github.com/edx/course-discovery/pull/1145/files#diff-651fda19c2550b81fbdc3c7de19a0dbbR168).